### PR TITLE
Add string-02

### PR DESCRIPTION
**Pull request by luuhieu8720**

### CODE REVIEW

File: DotnetIntern2021/Strings/Exercise2.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public class Exercise2
+    {
+        public string Ex2(string snakeCase)
+        {
+            string upperCase = "";
+            upperCase += snakeCase[0].ToString().ToUpper();
+            int i = 1;
+            while(i < snakeCase.Length)
+            {
+                if (string.Equals(snakeCase[i].ToString(), "_"))
```
Comment (voquanghoa) [inline DotnetIntern2021/Strings/Exercise2.cs]:
```suggestion
                if (snakeCase[i] == '_')
```

Comment (luuhieu8720) [reply DotnetIntern2021/Strings/Exercise2.cs]: done

File: DotnetIntern2021/Strings/Exercise2.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            upperCase += snakeCase[0].ToString().ToUpper();
+            int i = 1;
+            while(i < snakeCase.Length)
+            {
+                if (snakeCase[i] == '_')
+                {
+                    upperCase += snakeCase[i + 1].ToString().ToUpper();
+                    i++;
+                }
+                else upperCase += snakeCase[i];
```
Comment (voquanghoa) [inline DotnetIntern2021/Strings/Exercise2.cs]:
```suggestion
                else 
                {
                    upperCase += snakeCase[i];
                }
```

File: DotnetIntern2021/Strings/Exercise2.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        public string Ex2(string snakeCase)
+        {
+            string upperCase = "";
+            upperCase += snakeCase[0].ToString().ToUpper();
+            int i = 1;
+            while(i < snakeCase.Length)
+            {
+                if (snakeCase[i] == '_')
+                {
+                    upperCase += snakeCase[i + 1].ToString().ToUpper();
```
Comment (voquanghoa) [inline DotnetIntern2021/Strings/Exercise2.cs]:
```suggestion
                    upperCase += char.ToUpper(snakeCase[i + 1]);
```

File: DotnetIntern2021/Strings/Exercise2.cs [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+using System.Threading.Tasks;
+
+namespace DotnetIntern2021.Strings
+{
+    public class Exercise2
+    {
+        public string Ex2(string snakeCase)
+        {
+            string upperCase = "";
+            upperCase += snakeCase[0].ToString().ToUpper();
```
Comment (voquanghoa) [inline DotnetIntern2021/Strings/Exercise2.cs]: Dùng StringBuilder

File: DotnetIntern2021/Strings/Exercise2.cs (hunk 2 above)
Comment (luuhieu8720) [reply DotnetIntern2021/Strings/Exercise2.cs]: done

File: DotnetIntern2021/Strings/Exercise2.cs (hunk 3 above)
Comment (luuhieu8720) [reply DotnetIntern2021/Strings/Exercise2.cs]: done

File: DotnetIntern2021/Strings/Exercise2.cs (hunk 4 above)
Comment (luuhieu8720) [reply DotnetIntern2021/Strings/Exercise2.cs]: done

**Pull request merged**